src/box/lua/console.lua

```diff
@@ -417,14 +420,68 @@ local function table_pack(...)
     return {n = select('#', ...), ...}
 end
 
+local initial_env
+
+-- Get initial console environment.
+--
+-- If the console_session_scope_vars compat option is set to new,
+-- each console session has its own variable scope (environment)
+-- created before execution of a first command.
+--
+-- This environment initially contains a couple of frequently used
+-- built-in modules. A non-local variable assignment within a
+-- command modifies the environment.
+--
+-- The initial environment is returned by console.initial_env().
+-- It is a table that may be modified by a user or completely
+-- replaced with a new table using console.set_initial_env().
+-- These calls don't affect existing console sessions, only ones
+-- that are created afterwards.
+function M.initial_env()
+    if initial_env == nil then
+        initial_env = {
```

Comment:
What do you think about adding `ffi` by default too?
It is not about the user experience but more about the developer experience. Sometimes it is useful to check some `ffi` triks in the console.

Reply:
No problems, added.